### DISCUSSION TITLE
How to fetch user rating (i.e. IMDB Rating) for a movie?

### DISCUSSION THREAD

**Discussion by zvanjak** [cinemagoer/cinemagoer#381] [General]
I am a total newbie here, and I am actually surprised how easy it was to find a movie with the given title using Cinemagoer, and then fetch a bunch of data about it ... but, I am struggling with getting the rating number. 
I am probably missing something basic :(.

**Comment by alberanid** [comment]
hi @zvanjak , please always provide a minimal example of code.

As a general example to get the rating and a list of all available information for a given movie:
```python

#!/usr/bin/env python3

from imdb import IMDb

ia = IMDb()
search_results = ia.search_movie('Matrix')
# get the first result; it will only contains basic info like title and production year
the_matrix = search_results[0]
# fetch complete information about the title
ia.update(the_matrix)
# print rating
print(the_matrix.get('rating'))
# print all available info
print(sorted(the_matrix.keys()))
```

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
